biz.aQute.bndlib/src/aQute/bnd/build/model/EE.java

```diff
@@ -99,6 +101,15 @@ public Version getCapabilityVersion() {
 		return capabilityVersion;
 	}
 
+	public static EE highestFromVersion(String versionString) {
+		Version version = new Version(versionString);
+		return Arrays.stream(values())
+			.sorted(Collections.reverseOrder())
+			.filter(ee -> ee.capabilityVersion.compareTo(version) == 0)
```

Comment:
You should probably filter first before you sort as the filter may reduce to a single element which is easy to sort :-)